moma_controllers/path_admittance_controller/src/path_admittance_controller.cpp

```diff
@@ -135,41 +136,39 @@ void PathAdmittanceController::update(const ros::Time& time, const ros::Duration
   Eigen::Matrix3d R(q);
 
   // Update measured wrench and tracking errors
-  force_ext_ = Eigen::Vector3d(wrench_.wrench.force.x,
-                                 wrench_.wrench.force.y, 
-                                 wrench_.wrench.force.z);
+  force_ext_ = Eigen::Vector3d(!wrench_inverted_ ? wrench_.wrench.force.x : -wrench_.wrench.force.x,
+                               !wrench_inverted_ ? wrench_.wrench.force.y : -wrench_.wrench.force.y,
+                               !wrench_inverted_ ? wrench_.wrench.force.z : -wrench_.wrench.force.z);
```

Comment:
Maybe cleaner
```
force_ext_ = Eigen::Vector3d(wrench_.wrench.force.x, 
                             wrench_.wrench.force.y, 
                             wrench_.wrench.force.z)
if (wrench_inverted_) force_ext_ = -1.0 * force_ext_;
```

Reply:
True, I was already pre-optimizing (avoiding floating point multiplications in this case and boiling it down to simpler jump assembly instructions) but it's less readable like this. Probably a unary minus `force_ext_ = -force_ext_;` is even better.